From 8ae839e458a47b4c15125e28aed9f902001514e7 Mon Sep 17 00:00:00 2001
From: Joe Slater <joe.slater@windriver.com>
Date: Tue, 31 May 2022 12:11:06 -0400
Subject: [PATCH] expat.spec: use tis Release format

Signed-off-by: Joe Slater <joe.slater@windriver.com>
---
 SPECS/expat.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SPECS/expat.spec b/SPECS/expat.spec
index f2f860e..e8a3a09 100644
--- a/SPECS/expat.spec
+++ b/SPECS/expat.spec
@@ -1,7 +1,7 @@
 Summary: An XML parser library
 Name: expat
 Version: 2.1.0
-Release: 14%{?dist}
+Release: 14.el7_9%{?_tis_dist}.%{tis_patch_ver}
 Group: System Environment/Libraries
 Source: http://downloads.sourceforge.net/expat/expat-%{version}.tar.gz
 Patch0: expat-2.1.0-xmlwfargs.patch
-- 
2.29.2

