From 7970200c8d5781152bd7ca40ac99fb8c2954381e Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Thu, 23 Sep 2021 14:04:14 -0400
Subject: [PATCH] syscall: extend skip criteria for TestAmbientCapsUserns

Signed-off-by: Scott Little <scott.little@windriver.com>
---
 src/syscall/exec_linux_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/syscall/exec_linux_test.go b/src/syscall/exec_linux_test.go
index cc2140f..acc9b12 100644
--- a/src/syscall/exec_linux_test.go
+++ b/src/syscall/exec_linux_test.go
@@ -573,6 +573,7 @@ func TestAmbientCaps(t *testing.T) {
 }
 
 func TestAmbientCapsUserns(t *testing.T) {
+        checkUserNS(t)
 	testAmbientCaps(t, true)
 }
 
-- 
2.29.2

